src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data.tag;
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+/*
+ * Represents the taggings of the people added and deleted in the order
+ */
+public class Tagging {
+	
+	private String name;
+	private String tag;
+	private String command;
+	private static ArrayList<String> listOfTaggings = new ArrayList<String>();
+	
+	public Tagging(Person person, Tag tag, String command){
+			this.name = person.getName().toString();
```

Comment:
- indentation is off?

- I can't agree that this is an association class. Why? Because you are not maintaining the Person and Tag classes in Tagging.java. 
Hints: instead of maintaining the Strings name, tag, have Person and Tag classes. You can have getters for them too. 

- A choice of String variable to maintained the "command" is not good. consider a boolean or an enum